components/root/root.jsx

```diff
@@ -300,12 +304,8 @@ export default class Root extends React.Component {
                         component={TermsOfService}
                     />
                     <Route
-                        path={'/get_ios_app'}
-                        component={GetIosApp}
-                    />
-                    <Route
-                        path={'/get_android_app'}
-                        component={GetAndroidApp}
+                        path={'/vault'}
```

Comment:
Curious, why `/vault`?

Reply:
It was the route name picked by the original PR. I'm not sure if there's any significance to it or not.

Reply:
Can we go with something like `/landing` to be more clear?